### PR TITLE
Add autoconsent settings

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/1125189844152671/1202811961217167

### Description
This task adds the autoconsent settings screen
### Steps to test this PR

- [x] Launch the app
- [x] Go to settings -> Manage Cookie pop-ups
- [x] Setting should be disabled
- [x] Go to SERP and on the onboarding dialog click on hide and hide tips forever to avoid them clash with autoconsent.
- [x] Go to `https://privacy-test-pages.glitch.me/features/autoconsent`
- [x] You should see the autoconsent popup
- [x] Click on `Manage Cookie Pop-ups`
- [x] Go to settings -> manage cookie pop-ups
- [x] Setting should be enabled
- [x] Go to `https://privacy-test-pages.glitch.me/features/autoconsent` and refresh the page
- [x] You should get a toast message saying the cookie popup was handled and the page should have a button saying `I was clicked`
- [x] Go to settings -> manage cookie pop-ups and disable the setting
- [x] Go to `https://privacy-test-pages.glitch.me/features/autoconsent` and refresh the page
- [x] You should not see the previous toast message and the website should not have a button saying `I was clicked`
- [x] Clear data & cache
- [x] Launch the app
- [x] Go to SERP and on the onboarding dialog click on hide and hide tips forever to avoid them clash with autoconsent.
- [x] Go to `https://privacy-test-pages.glitch.me/features/autoconsent`
- [x] This time click on `No thanks`
- [x] Go to settings -> manage cookie pop-ups
- [x] Setting should be disabled

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2128** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2128" target="_blank">Graphite</a> 
    * **PR #2129** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2129" target="_blank">Graphite</a> 
    * **PR #2155** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2155" target="_blank">Graphite</a> 
    * **PR #2178** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2178" target="_blank">Graphite</a> 
    * **PR #2181** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2181" target="_blank">Graphite</a> 
      * **PR #2188** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2188" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/android/2188?utm_source=stack-comment).

File: autoconsent/autoconsent-impl/src/main/res/values/strings-autoconsent.xml [hunk 1]
```diff
@@ -15,7 +14,8 @@
...
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.duckduckgo.autoconsent.api">
-
-</manifest>
\ No newline at end of file
+<resources>
+    <string name="autoconsentTitle">Manage Cookie Pop-ups</string>
+    <string name="autoconsentDescription">When DuckDuckGo detects cookie consent pop-ups on sites you visit, we can automatically set your cookie preferences to minimize cookies and maximize privacy, then close the pop-ups.</string>
+    <string name="autoconsentToggle">Let DuckDuckGo manage cookie consent pop-ups</string>
+</resources>
```
Comment (aitorvs) [inline autoconsent/autoconsent-impl/src/main/res/values/strings-autoconsent.xml:21]: I guess these will be translated in separate PR (?)

File: autoconsent/autoconsent-store/build.gradle [hunk 2]
```diff
@@ -22,3 +22,7 @@ dependencies {
         exclude group: "org.jetbrains.kotlinx", module: "kotlinx-coroutines-debug"
     }
 }
+
+android {
+    namespace 'com.duckduckgo.autoconsent.store'
+}
```
Comment (aitorvs) [inline autoconsent/autoconsent-store/build.gradle:28]: nice, for the win

**Review by aitorvs — APPROVED** [review]
LGTM!

File: autoconsent/autoconsent-impl/src/main/res/values/strings-autoconsent.xml (hunk 1 above)
Comment (marcosholgado) [reply autoconsent/autoconsent-impl/src/main/res/values/strings-autoconsent.xml:21]: Yup
